packages/iris-grid/src/AdvancedFilterCreator.test.tsx

```diff
@@ -54,7 +54,7 @@ function makeAdvancedFilterCreatorWrapper(
       selectedValues: [],
     },
     model: irisGridTestUtils.makeModel(),
-    column: irisGridTestUtils.makeColumn(),
+    column: irisGridTestUtils.makeColumn('0'), // needs to match column name in makeModel so that columnIndex can be found when rendering
```

Comment:
There's probably a more fool-proof way to achieve this?

Reply:
Maybe I should be handling that null/undefined case differently in AdvancedFilterCreator.tsx

Reply:
I think this is fine. Anything else you'd do would be a bit long-winded and unnecessary. You could define the columns beforehand, for example, so something like:
```
const columns = irisGridTestUtils.makeColumns(3, 'test');
...
function makeAdvancedFilterCreatorWrapper(
....
    } = {
        model: irisGridTestUtils.makeModel(irisGridTestUtils.makeTable(columns)),
        column: column[0],
```

There's still a lot of tests here disabled, we should really look at re-enabling them.